sdk/provisioning/Azure.Provisioning.AppConfiguration/src/Azure.Provisioning.AppConfiguration.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains the core functionality for defining Azure infrastructure with dotnet code.</Description>
+    <Version>0.1.0-beta.1</Version>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net461;net472;net6.0</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);AZC0001</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager.AppConfiguration" />
```

Comment:
Do we want ProjectReferences now that we're in main?  I'm not sure if that'd be better or an unnecessary hassle.

Reply:
I don't think it will help